### PR TITLE
LPS-153224 | StructuredContentContainsPriorityField

### PR DESCRIPTION
**Pull request by Hijky**
**Background**
StructuredContent can be filtered with default priority

**Test Plan**
Given three web content is created with default priority
When with GET request I filter web content by the field priority with "/v1.0/sites/{siteId}/structured-contents"
Then you can see the web content filtered by the default in the body response

**JIRA Ticket**
https://issues.liferay.com/browse/LPS-153224

### CODE REVIEW

**Comment by Hijky** [comment]
ci:test:relevant

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:sf

**Comment by Hijky** [comment]
ci:test:relevant

**Review by magjed4289 — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/HeadlessDeliveryStructuredContent.testcase [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+			var responseToParse = HeadlessWebcontentAPI.createStructuredContentDraft(
+				data = "<p>My content</p>",
+				ddmStructureId = "${ddmStructureId}",
+				label = "Text",
+				name = "Text",
+				title = "MyTitle");
+		}
+
+		task ("When web contents are filtered with default priority") {
+			var responseToParse = HeadlessWebcontentAPI.filterStructuredContentWithPriority(filtervaule = "0.0");
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/HeadlessDeliveryStructuredContent.testcase:53]: typo, change to `filtervalue`

File: portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro [hunk 2]
```diff
@@ -139,4 +166,10 @@ definition {
 		return "${response}";
 	}
 
+	macro filterStructuredContentWithPriority {
+		var response = HeadlessWebcontentAPI._filterStructuredContentWithPriority(filtervaule = "${filtervaule}");
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro:170]: typo, change to `filtervalue`

File: portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro [hunk 3]
```diff
@@ -120,6 +120,33 @@ definition {
...
 
+	macro _filterStructuredContentWithPriority {
+		var portalURL = JSONCompany.getDefaultPortalURL();
+
+		var siteId = JSONGroupAPI._getGroupIdByName(
+			groupName = "Guest",
+			site = "true");
+
+		var curl = '''
+			${portalURL}/o/headless-admin-content/v1.0/sites/${siteId}/structured-contents?filter=priority%20eq%20${filtervaule} \
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro:131]: typo, change to `filtervalue`

**Comment by Hijky** [comment]
ci:test:sf

**Comment by magjed4289** [comment]
@magjed4289 test

**Review by magjed4289 — CHANGES_REQUESTED** [review]

**Comment by Hijky** [comment]
ci:test:sf

**Review by magjed4289 — APPROVED** [review]

**Comment by magjed4289** [comment]
ci:forward

**Pull request closed without merging**